src/caselawclient/models/documents.py

```diff
@@ -210,7 +210,10 @@ def jurisdiction(self) -> str:
 
     @property
     def court_and_jurisdiction(self) -> str:
```

Comment:
After sleeping on it, could we perhaps have something like
```suggestion
    def court_and_jurisdiction_identifier_string(self) -> str:
```
and a docstring explaining where a list of the valid codes can be found? That way it's clear that this is actually a coded representation, rather than a naive string for users.

Reply:
Actually a followup, is it worth creating an explicit new type for this with `NewType` like we do for `DocumentURIString`, maybe `CourtIdentifierString` so we can always be really explicit about what we're expecting to pass around?